sdk/containerregistry/azure-containerregistry/azure/containerregistry/_base_client.py

```diff
@@ -25,12 +25,9 @@ class ContainerRegistryApiVersion(str, Enum):
 class ContainerRegistryBaseClient(object):
     """Base class for ContainerRegistryClient and ContainerRepositoryClient
 
-    :param endpoint: Azure Container Registry endpoint
-    :type endpoint: str
-    :param credential: AAD Token for authenticating requests with Azure
-    :type credential: :class:`azure.identity.DefaultTokenCredential`
+    :param str endpoint: Azure Container Registry endpoint
+    :param :class:`azure.identity.DefaultTokenCredential` credential: AAD Token for authenticating requests with Azure
```

Comment:
Not sure how this one would render with Sphinx as a one-liner 🤔 I've never tried a one-liner with a class like this though, so I can't say for sure